src/include/sof/drivers/dw-dma.h

```diff
@@ -118,7 +118,7 @@
 #define DW_DMA_CFG_TRIES	10000
 
 /* channel drain timeout in microseconds */
-#define DW_DMA_TIMEOUT	1333
+#define DW_DMA_TIMEOUT	600
```

Comment:
what would happen if we reduced the tick to below 1ms, e.g. for Ultra-Low-Latency (ULL)
Should this become a variable that is set to 2/3 of the tick?

Reply:
Such a situation - with tick below 1ms - is already implemented in poll_for_register_delay()